sentry_sdk/integrations/graphene.py

```diff
@@ -53,7 +53,8 @@ def _sentry_patched_graphql_sync(schema, source, *args, **kwargs):
         with hub.configure_scope() as scope:
             scope.add_event_processor(_event_processor)
 
-        result = old_graphql_sync(schema, source, *args, **kwargs)
+        with hub.start_transaction(op="graphql", name=kwargs.get("operation_name")):
+            result = old_graphql_sync(schema, source, *args, **kwargs)
```

Comment:
We should be starting a span here rather than a transaction, since Graphene is likely to be called by a web server's response handler, which would already have a transaction automatically started for it (we integrate with the popular web frameworks, like Flask, Django, etc). Creating a transaction within a transaction leads to undefined behavior; they cannot be nested, so we need a span, not a transaction here.

Since spans do not have a `name`, the operation name should be stored in the span's `description` instead.

Lastly, the `op` should be one of the values listed under the GraphQL category in the "Usage" column of [this table](https://develop.sentry.dev/sdk/performance/span-operations/#web-server).